File: go/check_service.go
package main

import "errors"
import "test-service/spec"
import "cloud.google.com/go/civil"
import "github.com/google/uuid"
import "github.com/shopspring/decimal"

type CheckService struct{}

func (service *CheckService) CheckEmpty() (*spec.CheckEmptyResponse, error) {
	return nil, errors.New("implementation has not added yet")
}
func (service *CheckService) CheckQuery(pString string, pStringOpt *string, pStringArray []string, pDate civil.Date, pDateArray []civil.Date, pDatetime civil.DateTime, pInt int, pLong int64, pDecimal decimal.Decimal, pEnum spec.Choice, pStringDefaulted string) (*spec.CheckQueryResponse, error) {
	return nil, errors.New("implementation has not added yet")
}
func (service *CheckService) CheckUrlParams(intUrl int64, stringUrl string, floatUrl float32, boolUrl bool, uuidUrl uuid.UUID, decimalUrl decimal.Decimal, dateUrl civil.Date) (*spec.CheckUrlParamsResponse, error) {
	return nil, errors.New("implementation has not added yet")
}
func (service *CheckService) CheckForbidden() (*spec.CheckForbiddenResponse, error) {
	return nil, errors.New("implementation has not added yet")
}

File: java-spring/src/main/java/test_service/services_implementations/CheckService.java
package test_service.services_implementations;

import java.math.BigDecimal;
import java.time.*;
import java.util.*;

import org.springframework.stereotype.Service;

import test_service.models.*;
import test_service.services.check.*;

@Service("CheckService")
public class CheckService implements ICheckService {
	@Override
	public void checkEmpty() {
		return;
	}
	@Override
	public void checkQuery(String pString, String pStringOpt, String[] pStringArray, LocalDate pDate, LocalDate[] pDateArray, LocalDateTime pDatetime, int pInt, long pLong, BigDecimal pDecimal, Choice pEnum, String pStringDefaulted) {
		return;
	}
	@Override
	public void checkUrlParams(long intUrl, String stringUrl, float floatUrl, boolean boolUrl, UUID uuidUrl, BigDecimal decimalUrl, LocalDate dateUrl, Choice enumUrl) {
		return;
	}
	@Override
	public CheckForbiddenResponse checkForbidden() {
		return new CheckForbiddenResponseForbidden();
	}
	@Override
	public void sameOperationName() {
		return;
	}
}
